qiita_pet/templates/study_description_templates/sample_template_tab.html

```diff
@@ -9,7 +9,7 @@
       {% end %}
     </select>
     <br/><br/>
-    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a>
+    <a class="btn btn-primary" onclick="process_sample_template();">Process sample template</a> <a class="btn btn-primary" onclick="extend_sample_template();">Add samples to study</a>
```

Comment:
I just realized that if the dropdown menu has no files and you click any of these two buttons you'll get an error, while it is not problem introduced by this PR we certainly have to fix this for the buttons to be disabled unless there's a file selected.